### ISSUE TITLE
Switch language maintaining same regex functionality.

### ISSUE THREAD

**Issue by rosyth** [firasdib/Regex101#1629]
## Feature
For instance,  I can write in Perl  m/[^[:ascii:]]+/  to find any strings which have non-ascii characters.
But now I want to use the same construct in Python (without having to think too much), but Python (for reasons I can't fathom) chooses not to allow these constructs.
It would be nice to be able to have a button to  translate  a regex into language XYZ from another that we may be more familiar with.
Another place I often get caught out is working in Perl and then switching to VIM.

**Comment by firasdib** [comment]
I don't think this will be possible beyond rudimentary differences, such as `(?P<name>...)` being converted to `(?<name>...)`. How should things where there is no support be handled?

**Comment by Emasoft** [comment]
I strongly agree with this feature request. Most of the times we found a RegEx in the library that does what we want, but it is in the wrong flavour. 90% of the times it's in the wrong flavor. Especially the substitution string.
Automatic Flavor Conversion is an essenttial feature IMHO.
